Adding the "lipo" package

jdb78 opened the pull request:
<!--
Thank you very much for putting in this recipe PR!

This repository is very active, so if you need help with
a PR or once it's ready for review, please let the right people know.
There are language-specific teams for reviewing recipes.

Currently available teams are:
- python `@conda-forge/help-python`
- python/c hybrid `@conda-forge/help-python-c`
- r `@conda-forge/help-r`
- java `@conda-forge/help-java`
- nodejs `@conda-forge/help-nodejs`
- c/c++ `@conda-forge/help-c-cpp`
- perl `@conda-forge/help-perl`
- Julia `@conda-forge/help-julia`
- ruby `@conda-forge/help-ruby`

If your PR doesn't fall into those categories please contact
the full review team `@conda-forge/staged-recipes`.

Due to GitHub limitations first time contributors to conda-forge are unable
to ping these teams. You can [ping the team](https://conda-forge.org/docs/maintainer/infrastructure.html#conda-forge-admin-please-ping-team) using a special command in
a comment on the PR to get the attention of the `staged-recipes` team. You can
also consider asking on our [Gitter channel](https://gitter.im/conda-forge/conda-forge.github.io)
or on our [Keybase chat](https://keybase.io/team/condaforge.chat)
if your recipe isn't reviewed promptly.
-->

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipes```) and found some lint.

Here's what I've got...


For **recipes**:

 * The recipe must have some tests.
 * There are too few lines.  There should be one empty line at the end of the file.
 * license_file entry is missing, but is required.
 * `noarch` packages can't have selectors. If the selectors are necessary, please remove `noarch: python`.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipes```) and found it was in an excellent condition.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipes/lipo```) and found it was in an excellent condition.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipes```, ```recipes/lipo```) and found it was in an excellent condition.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipes/lipo```) and found it was in an excellent condition.

jdb78 commented:
This PR is ready for review @conda-forge-admin, please ping team

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-webservice.

I was asked to ping @conda-forge/staged-recipes and so here I am doing that.

synapticarbors commented on recipes/lipo/meta.yaml:
```diff
@@ -0,0 +1,49 @@
+  host:
+    - pip
+    - python
+    - scikit-learn >=0.22.1
```
With `poetry`, do you need anything beyond `poetry`, `pip` and `python`?

jdb78 replied:
true. fixed that

synapticarbors commented on recipes/lipo/meta.yaml:
```diff
@@ -0,0 +1,49 @@
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [py<36]
```
I think you can get rid of the `skip` and make this `noarch: python`:

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

and then just change `python` to `python >=3.6` under `host` and `run` below

jdb78 replied:
Right. Thanks for the pointer!

jdb78 commented:
Many thanks for the review and your commitment to conda-forge!! I addressed your comments and this PR is again ready for review @conda-forge-admin, please ping team

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-webservice.

I was asked to ping @conda-forge/staged-recipes and so here I am doing that.

synapticarbors commented on recipes/lipo/meta.yaml:
```diff
@@ -0,0 +1,46 @@
+    - python >=3.6
+    - poetry >=1.0.5
+  run:
+    - python
```
Can you also change this to `python >=3.6`

jdb78 replied:
Overseen this. Done.

jdb78 commented:
Thanks! I had overseen the missing python dependency specification. The PR is again ready for review @conda-forge-admin, please ping team

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-webservice.

I was asked to ping @conda-forge/staged-recipes and so here I am doing that.

synapticarbors commented:
@jdb78 -- thank you for your contribution to conda-forge. Once the feedstock for this package is created, please don't hesitate to contact us if you run into issues.
